### DISCUSSION TITLE
Discrepancies between local and remote implementation

### DISCUSSION THREAD

**Discussion by quetool** [anuraghazra/github-readme-stats#3296] [Q&A] [answered]
Hello! This is weird and not even sure how to ask but I'm noticing that Total Commits count is not being updated then on Github but when I clone the repo it's being shown properly. What could be the reason?

This is what is shown when cloning the repo and running it locally:
![local](https://github.com/anuraghazra/github-readme-stats/assets/14978705/74c9dcf8-bdb9-4278-8069-88eed776f361)

This is what is shown on my GH profile (check here https://github.com/quetool)
![remote](https://github.com/anuraghazra/github-readme-stats/assets/14978705/b9fc6cc5-61c4-489a-bd9e-21caf756fc4a)

**Comment by qwerty541** [comment, marked as answer]
Hey, @quetool! I assume that your problem related to caching, currently stats card in your profile shows updated info. Stats card update time depends from `cache_seconds` query option. By default it was set to 21600 seconds ( 6 hours ). You can set custom value from 21600 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options. You are also able to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) and set CACHE_SECONDS environment variable, this makes it possible to set a cache seconds without upper and lower level restrictions.

**Comment by quetool** [comment]
Hello @qwerty541! Thanks for your answer! I know all of that, I opened this discussion indeed because the data displayed on my GH profile wasn't being updated since weeks. I have resolved now by pushing an empty README file to my repo and then pushing again the code. That solved it, now the amounts are correct so for sure something related to cache but neither using my own Vercel instance was solving my issue.

Feel free to close this convo! Thanks!
